packages/mdctl-core/streams/section.js

```diff
@@ -40,6 +40,18 @@ const _ = require('lodash'),
       SectionsCreated = [],
       { privatesAccessor } = require('@medable/mdctl-core-utils/privates')
 
+let hashCode = function (str, seed = 0) {
```

Comment:
you can add this function mdctl-node-utils package https://github.com/Medable/mdctl/blob/master/packages/mdctl-node-utils/crypto.js and use crypto that already exists.